packages/create-app/bin/bundle.js

```diff
@@ -0,0 +1,40 @@
+import cleanBundledDependencies from '../../../bin/bundling/clean-bundled-dependencies.js'
+import ShopifyStacktraceyPlugin from '../../../bin/bundling/esbuild-plugin-stacktracey.js'
+import {build as esBuild} from 'esbuild'
+import glob from 'fast-glob'
+import {copy} from 'esbuild-plugin-copy'
+
+const external = [
+  // react-devtools-core is a dev dependency, no need to bundle it but throws errors if not included here.
+  'react-devtools-core',
+]
+
+// yoga wasm file is not bundled by esbuild, so we need to copy it manually
+const yogafile = glob.sync('../../node_modules/.pnpm/**/yoga.wasm')[0]
+
+esBuild({
```

Comment:
If I change the big CLI bundle script, I really need to change this one too (e.g. maybe we realise some extra plugin is needed)

I wonder if there is a way to organise these to minimise the chance of missing one of them?

Reply:
If we added a shared script, we will still need specific config for each package (extra plugins, defines, external dependencies) so we end up with 3 script files and that is more risk of editing the wrong one maybe?

Not sure though. The create-app one just needs to solve stuff that comes from `cli-kit` (it's only dependency)